backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -591,19 +625,37 @@ void SdlGraphicsManager::renderImGui() {
 		_imGuiInited = true;
 	}
 
-	ImGui_ImplOpenGL3_NewFrame();
+#ifdef USE_IMGUI_SDLRENDERER2
+	if (_imGuiSDLRenderer) {
+		ImGui_ImplSDLRenderer2_NewFrame();
+	} else
```

Comment:
shouldn't it be 

```C++
#ifdef USE_IMGUI_SDLRENDERER2
	if (_imGuiSDLRenderer) {
        ImGui_ImplOpenGL3_NewFrame();
     } else {
#endif
#ifdef USE_OPENGL
        ImGui_ImplOpenGL3_NewFrame();
#endif
#ifdef USE_IMGUI_SDLRENDERER2
     }
#endif
     ImGui_ImplSDL2_NewFrame();
```

Reply:
Indeed. it doesn't work if USE_OPENGL is not defined.
I applied your suggestion although I find it a bit heavy.
The alternative is to remove curly braces but I don't like that either.